modules/rosa-sts-aws-provisioned.adoc

```diff
@@ -0,0 +1,112 @@
+[id="rosa-sts-aws-policy-provisioned_{context}"]
```

Comment:
I don't see any information in this document that's unique to STS mode. This raises a maintenance / duplication concern for me. Is there a way to de-duplicate this in a future release of docs?

Reply:
I need to remove this file. It was created when we weren't sure whether there might be STS changes. Sorry about that!

Reply:
File is removed